notebooks/examples/Identify Inspected Buildings.ipynb

```diff
@@ -0,0 +1,279 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Import libraries and connect to the GIS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "import arcgis\n",
+    "from datetime import datetime\n",
+    "\n",
+    "gis = arcgis.gis.GIS(\"https://dev0007109.esri.com/portal\", \"tracker_analysis\", verify_cert=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Fetch the building footprints\n",
+    "\n",
+    "The building footprints are stored in a hosted feature layer. Let's query them and create a spatially enabled dataframe. We are using UTM Zone 19 as the spatial reference in order to have more accurate results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sr ={'wkid': 32619, 'latestWkid': 32619}  # UTM Zone 19N\n",
+    "buildings_item = gis.content.get(\"c8aa37b7378641e79a0442b15efb359a\")\n",
+    "buildings_df = buildings_item.layers[0].query(as_df=True, out_sr=sr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a map and plot the buildings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<img src=></img>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "webmap = gis.map(\"150 Commercial Street, Portland, ME\", zoomlevel=18)\n",
+    "webmap.add_layer(buildings_df)\n",
+    "webmap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Overlay some track data\n",
+    "Let's load some track data to visualize the problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<img src=></img>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "start_time = datetime(2020, 2, 1)\n",
+    "end_time = datetime(2020, 2, 2)\n",
+    "where_clause = f\"location_timestamp >= timestamp '{start_time.strftime('%Y-%m-%d %H:%M:%S')}' AND location_timestamp < timestamp '{end_time.strftime('%Y-%m-%d %H:%M:%S')}'\"\n",
+    "tracks_layer = arcgis.features.FeatureLayer(f'{gis.properties.helperServices[\"locationTracking\"][\"url\"]}/0', gis)\n",
+    "tracks_layer.filter = where_clause\n",
+    "webmap.add_layer(tracks_layer)\n",
```

Comment:
Can we use a unique webmap object here, rather than the one we just used? Can run into this fun bug

![flash screen](https://user-images.githubusercontent.com/14812213/74271808-861fc800-4cdb-11ea-950b-a8d314eee7a5.gif)

Reply:
Logged at: https://github.com/ArcGIS/geosaurus/issues/3205

Reply:
Thanks for loggin that. Yeah this is annoying. I saw this and chose to ignore it for now. Probably will have to have separate webmap objects, or only show the webmap once or something.

Reply:
This has been resolved in the latest master

Reply:
Nice, I was using the same one, so that the map is updated and displayed as you progress through the notebook.